### ISSUE TITLE
Default time shown without adding any args?

### ISSUE THREAD

**Issue by geraltofrivia** [KELiON/cerebro-timezones#5]
Thanks for making this super convenient plugin. I think it'd be cool to have just local time without entering any location. I could for instance enter `time mycity` to get current time, but the same can perhaps be accomplished with just `time`.

**Comment by KELiON** [comment]
@geraltofrivia why would you use Cerebro to get current time? You always can see it in your system

**Comment by geraltofrivia** [comment]
Fair point. Unless you're using Deepin (Manjaro), and the taskbar bugs and stops showing up altogether.
Or, even on a good day, you have to take your mouse down all the way, as opposed to `Hotkey -> t -> tab`.

A minor inconvenience at best.
